### PR TITLE
Fix trashbin scrolling

### LINKED ISSUE #3329: Scrolling in the trash bin is not possible if there are many deleted items.
The items at the end of the list just get cut off and the footer is not visible

### PR DESCRIPTION
**Pull request by GretaD**
fixes #3329

### CODE REVIEW

File: src/components/AppNavigation/CalendarList/Trashbin.vue [hunk 1]
```diff
@@ -236,6 +236,11 @@ export default {
 	max-width: 40vw;
 	margin: 2vw;
 }
+::v-deep .modal-wrapper .modal-container {
```
Comment (st3iny) [inline src/components/AppNavigation/CalendarList/Trashbin.vue:239]:
```suggestion

::v-deep .modal-wrapper .modal-container {
```

Make linter and CI happy

File: src/components/AppNavigation/CalendarList/Trashbin.vue [hunk 2]
```diff
@@ -236,6 +236,11 @@ export default {
...
+	overflow: scroll;
```
Comment (st3iny) [inline src/components/AppNavigation/CalendarList/Trashbin.vue:240]:
```suggestion
```

Not required because it is overwritten by the following 2 rules.

**Review by st3iny — APPROVED** [review]
Tested and scrolling works again.

**Comment by st3iny** [comment]
I rebased but did not change anything. Apparently, codecov went crazy on the last CI run.

**Comment by ChristophWurst** [comment]
> The backport to stable2.3 failed. Please do this backport manually.

@GretaD :wink:

**Pull request merged**